src/Ardalis.ApiCaller/HttpService.cs

```diff
@@ -303,7 +303,8 @@ public async Task<bool> HttpPutBytesWithoutResponseAsync(string uri, byte[] data
 
     private StringContent ToJson(object obj)
     {
-      return new StringContent(JsonSerializer.Serialize(obj), Encoding.UTF8, "application/json");
+      var json = JsonSerializer.Serialize(obj);
+      return new StringContent(json, Encoding.UTF8, "application/json");
```

Comment:
I did make a separate project for this, to use for a NuGet package. If you're making updates we should probably do them in that package as well (and I need to get it published to NuGet).